li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -72,6 +73,12 @@
       "full or the incomplete message has expired. The consumer will throw a LargeMessageDroppedException if this " +
       "configuration is set to true. Otherwise the consumer will drop the message silently.";
 
+  private static final String EXCEPTION_ON_INVALID_OFFSET_DOC = "If true, even when auto.offset.reset is set to " +
+      "earliest/latest/liclosest, InvalidOffsetException will be thrown to user upon calling to poll. This is to allow " +
+      "XinfraClients to properly invoke user callback upon InvalidOffsetException while handling the reset and corresponding " +
```

Comment:
should we avoid mention "XinfraClients" in the doc, since "XinfraClients" is not open-sourced yet?